cohttp.install

```diff
@@ -1,7 +1,16 @@
 bin: [
-  "?cohttp-server-async"
-  "?cohttp-server-lwt"
-  "?cohttp-curl-async"
-  "?cohttp-curl-lwt"
-  "?cohttp-proxy-lwt"
+  "?cohttp_server_async.native" { "cohttp-server-async" }
+  "?cohttp_server_async.byte"   { "cohttp-server-async" }
```

Comment:
Maybe the order matters but `_oasis` is targeting `best` so only one of these should exist.